lib/modules/apostrophe-schemas/index.js

```diff
@@ -1455,16 +1455,16 @@ module.exports = {
           },
           safeFor: 'manage',
           launder: function(value) {
-            if ((typeof field.choices) === 'string') {
-              field.choices = self.apos.modules[field.moduleName][field.choices](cursor.get('req'), field);
-            }
             // Support one or many
             if (Array.isArray(value)) {
               return _.map(value, function(v) {
```

Comment:
What about the plural case?

Reply:
Is it about the 2nd commit? I am not sure to understand your comment